samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -73,6 +73,8 @@
   static final String INMEMORY_KV_STORAGE_ENGINE_FACTORY =
       "org.apache.samza.storage.kv.inmemory.InMemoryKeyValueStorageEngineFactory";
 
+  public static final String CLEAN_LOGGED_STOREDIRS_ON_INIT = STORE_PREFIX + "init.clean";
```

Comment:
Shouldn't be init, should say container start or something similar.

Reply:
changed it to stores.container.start.clean 
how does that sound

Reply:
Maybe also make it per store.
`stores.<store-name>.clear.on.container.start`?

Reply:
done

Reply:
+1. `clearn.on.container.start` sounds more readable.